extensions/spark/build.sbt

```diff
@@ -0,0 +1,16 @@
+name := "spark"
+
+version := "0.1"
+
+scalaVersion := "2.12.10"
+scalacOptions += "-target:jvm-1.8"
+
+resolvers += Resolver.jcenterRepo
+
+libraryDependencies += "org.apache.spark" %% "spark-core" % "3.0.1"
```

Comment:
any reason we are using 3.0.1? Do we also want to put this into gradle.properties?

Reply:
OK, put into gradle.properties.